### DISCUSSION TITLE
High DPI Support

### DISCUSSION THREAD

**Discussion by antoniomiranda90** [ciderapp/Cider#1534] [Feature Request]
Hi!

First: Love the app! It feels amazing!

Second: Is it possible to have a "zoom" feature or a scalable UI? 

I'm running on a 4k display and gets a bit tiny... just a bit

Sorry if this is a duplicate request! I've searched and I didn't find any request like this.

Thank you!!

**Comment by booploops** [comment]
> Hi!
> 
> First: Love the app! It feels amazing!
> 
> Second: Is it possible to have a "zoom" feature or a scalable UI?
> 
> I'm running on a 4k display and gets a bit tiny... just a bit
> 
> Sorry if this is a duplicate request! I've searched and I didn't find any request like this.
> 
> Thank you!!

Under Settings > General> Keybindings there are bindings for zooming available
![image](https://user-images.githubusercontent.com/49113086/197939318-421850f5-7116-4d4c-81cd-59e4288cb0f6.png)

**Reply by antoniomiranda90** [reply]
It's not very easy to find, but problem solved!
Thank you
